### PR TITLE
Improves style for both block and inline code

### PR DESCRIPTION
**Pull request by zamith**
Inline code was not styled and at least my browser, because chrome is setting the `font-family` for the `code` tag, the block code was using the default monospace font.

Before:

screen shot 2016-03-15 at 4 47 23

After:

screen shot 2016-03-15 at 4 47 04

### CODE REVIEW

File: app/assets/stylesheets/components/_post.scss [hunk 1]
```diff
@@ -107,6 +121,15 @@ $Post-img-max-width: 800px + $Theme-spacing-small * 2;
...
       word-wrap: break-word;
     }
+
+    code {
+      @include font-properties(code, base, $Theme-font-sizes-extra);
+
+      padding-top: 0;
+      padding-bottom: 0;
+
+      background-color: transparent;
```
Comment (naps62) [inline app/assets/stylesheets/components/_post.scss]: why is this necessary?

Comment (zamith) [inline app/assets/stylesheets/components/_post.scss]: To override what we're setting on `code`. The thing is that there is no real good way of differentiating between inline and block code, other than block has a parent that is `.highlight`, so I set the style for the inline on `code` and undid some of it on `.highlight code`.

**Comment by naps62** [comment]
:+1:
